src/encoded/tests/test_audit_experiment.py

```diff
@@ -2185,6 +2185,8 @@ def test_audit_experiment_dnase_low_read_length(testapp,
                                                 'date_released': '2016-01-01',
                                                 'assay_term_name': 'DNase-seq'})
     res = testapp.get(base_experiment['@id'] + '@@index-data')
+    for error in collect_audit_errors(res):
+        print (error)
```

Comment:
should we get rid of this print statement in test code?  Or maybe it gets swallowed in stdout anyway...

Reply:
removed the print statement (sorry, I forgot to do it)